lib/adapter.js

```diff
@@ -145,9 +148,13 @@ class MochaAdapter {
     }
 
     emit (event, payload, err) {
+        // REVIEW - don't emit events for root test suite
+        if (payload.root) return
```

Comment:
I've added a comment - mocha fires one `suite:start` event, but two `suite:end` events for each suite. The second has `root = true`, and I'm ignoring it as it adds no value, and causes a mismatch between start and end counts.